### ISSUE TITLE
Using the new class in api 8

### ISSUE THREAD

**Issue by chaimleu** [rudsonlive/NavigationDrawer-MaterialDesign#11]
Can you make the new version of the class(1.0.3) working on api8?
I dont need the shadows that need api14 I am just need the NavigationDrawer.
Thanks and Good Evening
Haim.

**Comment by rudsonlive** [comment]
The first two versions have support api 8, look if they attend.
Unfortunately have no way to change the minSDK. I apologize for that.

Em qui, 22 de jan de 2015 13:05, chaimleu <notifications@github.com>
escreveu:

**Comment by chaimleu** [comment]
Thanks for the answer. I just need the custom header from the new version, but thanks for answering :+1:

**Comment by amouly** [comment]
I need to support from API 11, so maybe I will fork this project and remove the "strict" dependency of API14.

**Comment by rudsonlive** [comment]
Looking at the chart that Google itself offers do not see advantages in using an api less than 14.

Not to mention that the 11 does not even appear in the chart and that's why I put the 14 as MinSDK.

https://developer.android.com/about/dashboards/index.html

Thanks for the input: D

**Comment by amouly** [comment]
Sorry, I'm confused with the API numbers xD.

I need to support API 9 (2.3) or API 10 (Android 2.3.3). I understand what you said, but here in Argentina there is a LOT of Android devices running 2.X. That's sad, but the market continues selling new devices with that version.

**Comment by rudsonlive** [comment]
The biggest problem I found on the version where have made available the Header custom user was that the api 8-10 can only add a header before the list setAdapter.

ex:
Create header
mList.setAdapter ()
remove header

Already in version less than 14 I do not need to be mounting or removing my list whenever you want to use a header.

Outside this may limit me for future features.

Already here in Brazil for years that do not get work with api 8, always caught designs with your minSDK 14.

Once again I apologize friend.

**Comment by amouly** [comment]
Oh, now I understand. That's a huge problem, because the Header it's very important.

Maybe limiting the Header functionality only for API 14 >= projects? This can be done by a series of conditions on the code.

I have to investigate about this, maybe there is a solution for this ;).

**Comment by rudsonlive** [comment]
I got to make this feature work even in api 8+. What I found interesting is not having to create the list whenever Add & remove the header.

The developer has two options, either it use the default header or use a custom header.

So imagine the following situation:

The developer makes a custom header with a login button. In that case I would have to remove my custom header, to set zero in my adapter, set the user's default header and ride my list again.

I did not think very functional and not dedicated myself a lot of time it does not.

But if you find a solution that was really very good mute the api for 8+ today kkkkkkkkkkkkk

Thank U
